schema/contact.json

```diff
@@ -32,6 +32,16 @@
         ]
       }
     },
+   "externalIdentifiers": {
+      "type": "array",
+      "description": "Identifier for a contact, such as ORCid, RoR",
```

Comment:
@dwalt  Recommend using ORCID iD and ROR as per recommendations from maintaining organizations.
  - https://info.orcid.org/brand-guidelines/#the-orcid-name
  - https://ror.org/display-guidelines/#ror-glossary

Reply:
@hmaier-fws To clarify, the description would look like this: "Identifier for a contact, such as ORCID ID or ROR"?

Reply:
Yes, except "ORCID iD" (lowercase "i")